testscript/testdata/testscript_duplicate_name.txt

```diff
@@ -0,0 +1,34 @@
+# Check that RequireUniqueNames works;
+# it should reject txtar archives with duplicate names as defined by the host system.
+
+unquote scripts-case-insensitive/testscript.txt
+unquote scripts-case-sensitive/testscript.txt
+unquote scripts-normalized/testscript.txt
+
+testscript scripts-case-insensitive
+testscript scripts-case-sensitive
+testscript scripts-normalized
+
+[darwin] ! testscript -unique-names scripts-case-insensitive
+[darwin] stdout 'duplicate name'
+
+[linux] testscript -unique-names scripts-case-insensitive
+
+[windows] ! testscript -unique-names scripts-case-insensitive
+[windows] stdout 'duplicate name'
```

Comment:
Do Mac and Windows _always_ use case insensitive filesystems?

Reply:
We both know that they don't *always* :) It's rare, but possible, for macOS users to run their systems with case sensitive filesystems, for example, even though this tends to break a large number of macOS applications.

The effort of making the tests detect whether the filesystem they're running on is case sensitive or not is fairly large. I think you'd have to create a file with a long random lowercase name (to avoid conflicts with any existing files) and test if it exists with the name uppercased. You'd have to be careful to use the exact directory that the tests run in. You can't use a temporary directory as it by be mounted as a different filesystem.

So, for now, I've removed these parts of the tests.